### PR TITLE
[#50] Add Resources section to Java installation guide

### PR DESCRIPTION
**Pull request by itstrueitstrueitsrealitsreal**
Part of #50

Added links to the official SDKMAN documentation and a video installation guide, might be useful for students who want to delve deeper into the software or need a more visual approach to guide them through the process of installing the tool.

### CODE REVIEW

**Review by hjungwoo01 — COMMENTED** [review]
LGTM

**Review by hjungwoo01 — APPROVED** [review]
LGTM. The addition of these resources will be helpful.

File: tutorials/javaInstallationMac.md [hunk 1]
```diff
@@ -69,6 +69,14 @@ If you have multiple versions of Java installed, you can switch between them usi
   sdk default java <version>
   ```
 
+<!-- ======================================================================= -->
+
+## Resources
+
+* [SDKMAN Documentation](https://sdkman.io/)
+
+* [SDKMAN Video Installation Guide](https://www.youtube.com/watch?v=5cc8IZRvRcI&t=32s)
```
Comment (damithc) [inline tutorials/javaInstallationMac.md:78]:
```suggestion
* [SDKMAN Documentation](https://sdkman.io/)
* [SDKMAN Video Installation Guide](https://www.youtube.com/watch?v=5cc8IZRvRcI&t=32s)
```
Blank line is not needed

Comment (itstrueitstrueitsrealitsreal) [reply tutorials/javaInstallationMac.md]: Thanks for the review, rectified the issue.

**Comment by damithc** [comment]
@itstrueitstrueitsrealitsreal The first link is mentioned in the intro. I added the video at the relevant place in the main body in https://github.com/se-edu/guides/commit/fdc8a23cb00c767b177511cb749e0e8a31b47566

**Pull request closed without merging**